src/main/webapp/resources/js/pages/projects/linelist/components/Export/ExportDropdown.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
-
 import PropTypes from "prop-types";
 import { Button, Dropdown, Menu } from "antd";
+import {
+  DownloadOutlined,
+  FileExcelOutlined,
+  FileOutlined
+} from "@ant-design/icons";
 
 export function ExportDropDown(props) {
```

Comment:
This export dropdown button doesn't provide the user with the context that it is an actual dropdown. Maybe the download icon within can be replaced with arrow icons (facing up when button is hovered and facing down otherwise)?